components/utilities/object.js

```diff
@@ -0,0 +1,21 @@
+(function(d3, fc) {
```

Comment:
Yeah, the README says that the components are Firefox 3+ and Safari 3.2, but I don't think Object.Keys was supported until version 4 and 5 respectively.